enterprise/utils.py

```diff
@@ -455,20 +455,40 @@ def find_enroll_email_template(enterprise_customer, template_type):
     return enterprise_template_config
 
 
+def create_dict_from_user(user):
```

Comment:
Are you sure that the Django user object doesn't already have a serializer? Home writing one like this adds complexity. Can you look at
```

from django.forms.models import model_to_dict
model_to_dict(instance)
```
function and see if it meets your needs here?

Reply:
thanks, thinks I should be able to make that work and avoid a custom function, will be back!